core/math/aabb.h

```diff
@@ -81,6 +81,8 @@ class AABB {
 	_FORCE_INLINE_ bool has_point(const Vector3 &p_point) const;
 	_FORCE_INLINE_ Vector3 get_support(const Vector3 &p_normal) const;
 
+	Vector3 get_centre() const { return position + (size * 0.5); }
```

Comment:
It should be `get_center()` since we use US spelling.

Reply:
Done. :+1: